tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "strictPropertyInitialization": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
-    "esModuleInterop": false,
+    "esModuleInterop": true,
```

Comment:
It's always tricky to touch tsconfig. I would not touch it if possible

Reply:
I need `esModuleInterop` enabled to use the default import statement with secp256k1. This one only adds two functions/helpers: https://www.typescriptlang.org/tsconfig#esModuleInterop